### DISCUSSION TITLE
问问大家，大家的订阅日历能正常显示吗

### DISCUSSION THREAD

**Discussion by cng10001** [jxxghp/MoviePilot#2851] [问题咨询]
已经是最新版的1.9.17
不知道从什么时候开始日历不显示了，大家都正常吗
![image](https://github.com/user-attachments/assets/ecb5a77c-a97e-488e-9126-e2f950ea86bb)

**Comment by SouL87a** [comment]
一样

**Reply by cddjr** [reply]
设置页清一下缓存

**Reply by cng10001** [reply]
你用过迁移订阅插件吗

**Reply by SouL87a** [reply]
> 你用过迁移订阅插件吗

没用过，是从nt重新配置到mp的

**Comment by longfei12k** [comment]
插件清理下？还是哪里清理？

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月16日 09:56 |
| 收件人 | jxxghp/MoviePilot ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | Re: [jxxghp/MoviePilot] 问问大家，大家的订阅日历能正常显示吗 (Discussion #2851) |

设置页清一下缓存

**Comment by cng10001** [comment]
![image](https://github.com/user-attachments/assets/a93ac352-9bc4-4029-8c2f-9d89d248286f)
这个吗 么用哎
